src/Sfa.Tl.ResultsAndCertification.Application/Mappers/Converter/Specialism/SpecialismConverterBase.cs

```diff
@@ -13,16 +13,42 @@ protected static string GetSingleSpecialismProperty(IEnumerable<TqRegistrationSp
             return registrationSpecialism != null ? getPropertyValue(registrationSpecialism) : string.Empty;
         }
 
-        protected static string GetDualSpecialismProperty(IEnumerable<TqRegistrationSpecialism> specialisms, Func<TlDualSpecialism, string> getPropertyValue)
+        protected static string GetDualSpecialismNameProperty(IEnumerable<TqRegistrationSpecialism> specialisms, Func<TlDualSpecialism, string> getPropertyValue)
```

Comment:
There's no point in calling this method GetDualSpecialismName because the property we get back is specified by the getPropertyValue function